package.json

```diff
@@ -12,36 +12,35 @@
     "lint": "eslint --ext js,jsx ."
   },
   "dependencies": {
-    "@babel/runtime": "^7.1.2",
-    "bcrypt": "^1.0.1",
+    "@babel/runtime": "^7.1.5",
+    "bcrypt": "^3.0.2",
     "bootstrap": "^3.3.6",
     "classnames": "^2.2.5",
     "element-resize-detector": "^1.1.14",
     "express": "^4.14.0",
-    "googleapis": "^34.0.0",
-    "history": "^2.0.0",
```

Comment:
We weren't actually using the history library directly; it's also pulled in by react-router, which is how we use it.